tools/workbench/src/main/java/org/eclipse/rdf4j/workbench/base/TransformationServlet.java

```diff
@@ -31,6 +32,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@MultipartConfig
```

Comment:
I suspect this annotation is not inherited, and would have to be added to all subclasses. But hoping tests and/or maintainers can verify if this works.

Reply:
I'm not familiar with this annotation. What does it do?

Reply:
It adds multipart request support to the servlet. Can also be configured in web.xml. I am happy to do that instead, but I need to know which servlets need it - as this is configured per servlet. See https://docs.oracle.com/javaee/7/tutorial/servlets011.htm for additional details.